docs/configuring-socorro.rst

```diff
@@ -1,25 +1,68 @@
 .. index:: configuring-socorro
 
-Configuring Socorro
-===================
+Configure and run Socorro
+=========================
 
-Socorro produces graphs and reports, most are updated once per day.
+Initial setup
+-------------
 
-You must enter information about your releases into Socorro in order
-for this to work, and this information must match the incoming crashes.
+Socorro provides a setup script which attempts to initialize everything.
+If you have a distributed setup you should review this script and run
+the appropriate commands on the right servers::
+
+    sudo setup-socorro
+
+Start services
+--------------
+
+A full Socorro install requires the following to be running::
+
+    sudo systemctl enable socorro-collector
+    sudo systemctl enable socorro-processor
+    sudo systemctl enable socorro-middleware
+    sudo systemctl enable socorro-webapp
+
+Configure Nginx
+---------------
+
+Public-facing services like socorro-collector and socorro-webapp
+should be fronted by a webserver like Nginx.
 
-Configure Elasticsearch
------------------------
+For instance socorro-collector can be exposed via its uwsgi socket::
+    
+        location / {
+                uwsgi_pass unix:/var/run/uwsgi/socorro-collector.sock;
+                include uwsgi_params;
+        }
 
-Some key features of Socorro (like Super Search) depend on Elasticsearch.
-This service needs to be initialized before it can be used:
+The socorro-webapp service needs static files to be exposed as well::
 
-.. code-block:: bash
 
-    cd scripts && python ./setup_supersearch_app.py
+        location / {
+                uwsgi_pass unix:/var/run/uwsgi/socorro-webapp.sock;
+                include uwsgi_params;
+        }
 
-For more comprehensive information about our use of Elasticsearch, head to
-our :ref:`elasticsearch-chapter` documentation page.
+        location /static {
+                alias /data/socorro/webapp-django/static/;
+        }
+
+Cron jobs
+---------
+
+Socorro includes a crontab manager called Crontabber. This needs
```

Comment:
Includes or *uses*?  I'm not clear on whether crontabber is (to be) shipped with the socorro RPM or whether it needs to be installed alongside like any other dependency.